### ISSUE TITLE
[Keyboard Navigation - Intelligent support - IRIS]: When we navigate using keyboard the focus order of the Chat is incorrect in the IRIS bot chat.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4610]
**PWD Impact:**
Users who are dependent on keyboard will face difficulty if the focus order of the bot is incorrect.

**Repro steps:**
Step 1: Open the above URL in the browser
Step 2: Resource360 Dashboard page will be displayed.
Step 3: Now navigate through the header of the page and navigate to IRIS Bot button and invoke it
Step 4: IRISI bot chat popup will get opened
Step 5: Now navigate to the 'top questions for my role' button in the chat and press enter
Step 6: Now message containing options for top questions will be displayed, now press tab key to move focus to them and verify if the focus moves to the options or not.

**Actual Result:**
When we select 'top questions for my role' button, options for tops questions appears but focus doesn't move to that message. Then you have to press tab key or shift tab key 2-3 times and focus will appear on the last message sent from you For example - In this case when you select 'top questions for my role' button, top questions for my role message will be sent from our side and now when the options for top questions appears, you have to press tab/shift + tab 2-3 times and focus will land on top questions for my role message and not on the options for the top questions that appeared.

**Expected result:**
When we select 'top questions for my role' button, options for tops questions appears, now when we press tab focus should move to the options for top questions that appeared and not to the previous message.

**Observation:** The issue is similar through the IRIS bot chat.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EQnwpICBhyNEs80kSOr1v2IBQ1tiKQj6y90QKG1cRcwCEw?e=75BLFO

**Test environment:**
OS: Windows 11 22H2 (OS Build 22621.674)
Browsers: Edge Chromium (107.0.1418.26)
URL: https://servicesuat.microsoft.com/#/opdashboard/

**Comment by pgopinadhu** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS2.4.3;#Accessibility;#Win11-Edge(Chromium);#BotFrameworkWebChat-Jul22;

**Comment by compulim** [comment]
@pgopinadhu I couldn't see the bot using the URL with my internal credentials.

This is by design. Any buttons sent from the bot are apparently asynchronous (a.k.a. delayed). We cannot move focus to asynchronous widgets because that would interrupt screen reader reading live regions.
